d2l-floating-buttons.html

```diff
@@ -82,6 +82,11 @@
 			is: 'd2l-floating-buttons',
 
 			properties: {
+				alwaysFloat: {
+					reflectToAttribute: true,
+					type: Boolean,
+					value: false
```

Comment:
No worries.  It's mostly a consideration for high-use components.  Nevertheless, if it's not needed then might as well ditch it.